flutter_loggy/pubspec.lock

```diff
@@ -26,7 +26,7 @@ packages:
       name: loggy
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "2.0.0"
+    version: "2.0.0+1"
```

Comment:
Shouldn't this also be:
```suggestion
    version: "2.0.1"
```

Reply:
Yeah it should. But this is `.lock` file that is automatically generated, I don't feel comfortable changing those manually. My plan was to push `loggy` to pub, then update `flutter_loggy` to use new loggy version before pushing it to the pub.dev.